homeassistant/components/honeywell/climate.py

```diff
@@ -125,6 +126,10 @@ def __init__(self, data, cool_away_temp, heat_away_temp):
 
         _LOGGER.debug("latestData = %s ", data.device._data)
```

Comment:
This is existing, but we want to avoid side effects in `__init__`

Reply:
Removed.